sql/plan/create_index.go

```diff
@@ -116,6 +116,8 @@ func (c *CreateIndex) RowIter(ctx *sql.Context) (sql.RowIter, error) {
 		return nil, err
 	}
 
+	log := logrus.WithField("id", index.ID())
+
 	go func() {
 		err := driver.Save(ctx, index, iter)
```

Comment:
Instead of adding some custom logs to pilosa driver, we can wrap the IndexKeyValue iterator to have the same logs for all the drivers, WDYT?

Reply:
Sounds good. I thought about that, but adding another layer felt a bit too much for just logging, but I don't think the performance loss is gonna be huge here so it makes sense to do it.